From: Thomas Gleixner <tglx@linutronix.de>
Date: Thu, 24 May 2012 17:54:51 +0000
Subject: [PATCH] x86/mce: Cleanup timer mess
Patch-mainline: v3.5-rc2
Git-commit: 82f7af09e6fb58fb725c850d725d5e8780a9bec2
References: bnc#786450

Use unsigned long for dealing with jiffies not int. Rename the
callback to something sensible. Use __this_cpu_read/write for
accessing per cpu data.

Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
Acked-by: Borislav Petkov <borislav.petkov@amd.com>
Signed-off-by: Tony Luck <tony.luck@intel.com>
Acked-by: Michael Chang <mchang@suse.com>

Automatically created from "patches.arch/x86-mce-Cleanup-timer-mess.patch" by xen-port-patches.py

---
 arch/x86/kernel/cpu/mcheck/mce.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/x86/kernel/cpu/mcheck/mce.c
+++ b/arch/x86/kernel/cpu/mcheck/mce.c
@@ -1298,7 +1298,7 @@ void mce_log_therm_throt_event(__u64 sta
  * mechanism will be done in XEN for Intel CPUs
  */
 #if defined (CONFIG_X86_XEN_MCE)
-static int check_interval = 0; /* disable polling */
+static unsigned long check_interval = 0; /* disable polling */
 #else
 static unsigned long check_interval = INITIAL_CHECK_INTERVAL;
 #endif
